geometry_msgs/PoseStamped goal_pose
---
geometry_msgs/Quaternion goal_heading